tests/e2e/pubsub/pubsub_test.go

```diff
@@ -289,7 +289,7 @@ var pubsubTests = []struct {
 }
 
 func TestPubSub(t *testing.T) {
-	t.Log("Enter TestPubSub")
+	log.Println("Enter TestPubSub")
```

Comment:
why use log.Println?
t.Log can print file and line

Reply:
we are using log.Printf or log.Println in the rest of the file.